bridge.h

```diff
@@ -1,3 +1,9 @@
+/*
+ * SPDX-FileCopyrightText: 2020 - 2025 SAP SE
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 // SPDX-FileCopyrightText: 2020 SAP SE
 // SPDX-FileCopyrightText: 2021 SAP SE
 // SPDX-FileCopyrightText: 2022 SAP SE
```

Comment:
Leftovers

Reply:
This has still the leftovers or is my view bugging? :D

Reply:
Think this is still leftover :D I thought I got them all